ipcc/src/lib.rs

```diff
@@ -4,26 +4,28 @@
 
 // Copyright 2023 Oxide Computer Company
 
-//! Utilities for key/value pairs passed from the control plane to the SP
-//! (through MGS) to the host (through the host/SP uart) via IPCC.
+//! An interface to libipcc (inter-processor communications channel) which
+//! currently supports looking up values stored in the SP by key. These
+//! values are passed from the control plane to the SP (through MGS) or
+//! are maniuplated through the `ipcc` command found on Helios.
```

Comment:
Strictly speaking the `ipcc` command is just a test utility - it's not an interface and should not be used by anything other than an engineer at the CLI. The anonymous dtrace helper, as an example, use libipcc to set values.

```suggestion
//! An interface to libipcc (inter-processor communications channel) which
//! currently supports looking up values stored in the SP by key. These
//! values are variously static, passed from the control plane to the SP
//! (through MGS) or set from userland via libipcc.
```